Script.py

```
@@ -113,7 +112,7 @@ class script(object):
 ⭐ Rating: {rating} / 10</b>
 """
 
-    FILE_CAPTION = """<b>{file_name}\n\n𝖩𝗈𝗂𝗇➥ 「<a href="https://t.me/learningbots79">𝙊𝙧𝙞𝙜𝙞𝙣𝙖𝙡𝙨</a>」</b>"""
+    FILE_CAPTION = """<b>{file_name}\n\n𝖩𝗈𝗂𝗇➥ 「<a href="https://t.me/Mrshubh_𝟸𝟶𝟶𝟽">𝗦𝗨𝗣𝗣𝗢𝗥𝗧</a>」</b>"""
```

Ruff (0.13.2)

String contains ambiguous `𝖩` (MATHEMATICAL SANS-SERIF CAPITAL J). Did you mean `J` (LATIN CAPITAL LETTER J)?

(RUF001)